tests/extproc/extproc_test.go

```diff
@@ -50,6 +50,16 @@ func TestE2E(t *testing.T) {
 	openAIAPIKey := getEnvVarOrSkip(t, "TEST_OPENAI_API_KEY")
 	requireRunEnvoy(t, accessLogPath, openAIAPIKey)
 	configPath := t.TempDir() + "/extproc-config.yaml"
+
+	// Test with APIKey
```

Comment:
super nit (i would like any whole line comment ending with starting with a capital letter and ending with a period)
```suggestion
	// Test with APIKey.
```